web/src/main/java/no/nav/modiapersonoversikt/service/enhetligkodeverk/KodeverkConfig.kt

```diff
@@ -13,11 +13,31 @@ enum class KodeverkConfig(private val kilde: EnhetligKodeverk.Kilde) {
     DISKRESJONSKODER(FellesKodeverkKilde("Diskresjonskoder")),
     VALUTA(FellesKodeverkKilde("Valutaer")),
     TEMA(FellesKodeverkKilde("Tema")),
-    SF_TEMAGRUPPER(SfHenvendelseKodeverkKilde());
+    SF_TEMAGRUPPER(SfHenvendelseKodeverkKilde()),
+    NAVSKJEMAOGVEDLEGGSKODER(
```

Comment:
Brukes dette kodeverket noe sted?

Reply:
Nei, Ankur sin PR fjernet de delene av koden i `BulletproofKodeverkService` som brukte det lokale kodeverket. Så eneste stedet ellers det ble brukt var for å importere bønnen i `SakServiceConfig`. Jeg er usikker på om det er noe annet sted jeg skal bruke det kodeverket?